<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentFilterGUI</class>
 <widget class="QWidget" name="CommentFilterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CompactToolBar" name="toolBar" native="true"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionPowerOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/power_off.ico</normaloff>
     <normalon>:/icons/power_on.ico</normalon>:/icons/power_off.ico</iconset>
   </property>
   <property name="text">
    <string>Power on</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompactToolBar</class>
   <extends>QWidget</extends>
   <header>Editor/widgets/CompactToolBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Editor.qrc"/>
 </resources>
 <connections/>
</ui>
